third_party/CMakeLists.txt

```
@@ -1,9 +1,15 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
 
 project(torchaudio_third_parties)
-include(ExternalProject)
 
-set(INSTALL_DIR ${CMAKE_CURRENT_SOURCE_DIR}/install)
+option(BUILD_SOX "Build libsox statically")
+option(BUILD_TRANSDUCER "Build transducer")
+
+SET(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

nit: `SET` is capitalized but not the other cmake commands in this file :)

```suggestion
set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```

---

sure. I forgot to lowercase it.